setup.cfg

```diff
@@ -63,7 +63,7 @@ install_requires =
     typing_extensions
     toolz>=0.10.0
     tqdm>=4.56.0
-    vispy>=0.6.4
+    vispy>=0.7.1
```

Comment:
Do we need to bump the vispy requirement since we are vendoring the Volume visual?

Reply:
Good catch! Relic from earlier iterations :) thanks!

Reply:
Sorry, is the conclusion of this we don't need to bump and this should be reverted or we do need to bump and this is now correct?

Reply:
de-bumped!